docs/api-reference/carto/basemap.md

```diff
@@ -36,13 +36,13 @@ To use pre-bundled scripts:
 
 ```html
 <script src="https://unpkg.com/deck.gl@^8.2.0/dist.min.js"></script>
-<script src="https://unpkg.com/@deck.gl/carto@^8.2.0/dist.min.js"></script>
+<script src="https://unpkg.com/@deck.gl/carto@^8.5.0/dist.min.js"></script>
```

Comment:
Is there a reason for this variation on unpkg versions (8.0.0, 8.2.0, 8.5.0)?

Reply:
No, let's update all to 8.5.0